src/common/tusb_mcu.h

```diff
@@ -369,6 +369,10 @@
   #define TUP_DCD_ENDPOINT_MAX    7 // only 5 TX FIFO for endpoint IN
   #define CFG_TUSB_OS_INC_PATH_DEFAULT   freertos/
 
+  #if CFG_TUSB_MCU == OPT_MCU_ESP32S3
+    #define TUP_MCU_MULTIPLE_CORE 1
+  #endif
+
```

Comment:
@hathach I'm sorry for the late review, but could we add ESP32-P4 to the list of multiple core MCUs?

You can get the information also dynamically from `SOC_CPU_CORES_NUM` that can be included from `#include "soc/soc_caps.h"`

Reply:
No problems at all. We are all busy. Normally I am not in hurry, but this time I need to use this to fix other issue :)